### DISCUSSION TITLE
Delete a repository

### DISCUSSION THREAD

**Discussion by aliprogammerali** [community/community#43981] [New to GitHub]
Hello, I am new to Github. How delete my repository?

**Comment by melvincwng** [comment]
Hey @asimpledeveloper, first of all, allow me to welcome you to our GitHub Community 👋 

This is a place where you can feel free to ask either technical or non technical questions. We are all here to learn and grow together as developers.

With regards to your question on how to delete a repository,  may I suggest you refer to this [GitHub Documentation page](https://docs.github.com/en/repositories/creating-and-managing-repositories/deleting-a-repository) over here. 

It will contain the answer to your question.

Hope this helps 😃!

**Comment by skadomsky** [comment]
Click into your profile and find the repository you want to delete, go 'Settings'. Then scroll down, click on the 'Delete this repository' button, enter your password and press enter. All done.
